Source/control.cpp

```diff
@@ -52,13 +52,19 @@ std::optional<CelSprite> pSpellCels;
 BYTE sgbNextTalkSave;
 BYTE sgbTalkSavePos;
 
+/**
+ * @brief Set if the life flask needs to be redrawn during next frame
+*/

```

Comment:
The stars are not aligning:
```suggestion
/**
 * @brief Set if the life flask needs to be redrawn during next frame
 */
```

Reply:
Thanks for the pickup, Visual studio is a bit inconsistent about applying style rules and doesn't handle /** comments well, I'll see if I can fix it.